### ISSUE TITLE
SplitWithLines (python split_with_lines ) transfer source attributes?

### ISSUE THREAD

**Issue by fluviotect** [jblindsay/whitebox-tools#226]
Is it possible to transfer attributes from either (or, preferably, both) of the input files for split_with_lines? 

My output has none of the attributes. At the least, I would like to be able to transfer the respective ID fields from the input files.

**Comment by Afrancioni** [comment]
Hi @fluviotect, we will look into adding this functionality for the next stable release of WhiteboxTools.

**Comment by jblindsay** [comment]
Yes, I think that this is something that I could probably add to the tool for the next release.
